src/main/java/com/monitorjbl/xlsx/StreamingReader.java

```diff
@@ -111,11 +127,57 @@ private void handleEvent(XMLEvent event) throws SAXException {
         currentCell = new StreamingCell(CellReference.convertColStringToIndex(coord[0]), Integer.parseInt(coord[1]) - 1);
 
         Attribute type = startElement.getAttributeByName(new QName("t"));
+/*ForgetMeNot--ibell--20150911--old code which didn't determine numeric vs date data types.
         if(type != null) {
           currentCell.setType(type.getValue());
         }
+*/
+        //
+        // This block will determine the value type and set the appropriate
+        // XSSF data type (used to format numeric cell values).
+        //
+        final String typeValue = (type != null) ? type.getValue() : "";
```

Comment:
No problem.  As I indicated in the pull request, I offered it a s a temporary fix to a more elegant solution.  I have wrapped up the project in which I implemented this.  As such, I don't have the time to perfect the solution.  Feel free to use the submission as a suggestion and implement the functionality as you see fit.  I will monitor the repo to see when it is done so I can add the Maven dependency instead of having the monitorjbl classes in my project.  Thanks.